src/core/lib/iomgr/tcp_server_utils_posix_ifaddrs.cc

```diff
@@ -133,21 +133,25 @@ grpc_error_handle grpc_tcp_server_add_all_local_addrs(grpc_tcp_server* s,
       err = GRPC_ERROR_CREATE_FROM_STATIC_STRING("Failed to set port");
       break;
     }
-    std::string addr_str = grpc_sockaddr_to_string(&addr, false);
+    auto addr_str = grpc_sockaddr_to_string(&addr, false);
+    if (!addr_str.ok()) {
+      return GRPC_ERROR_CREATE_FROM_CPP_STRING(
+          std::string(addr_str.status().ToString()));
```

Comment:
No need to construct a new `std::string` here.

Reply:
Done.